pyproject.toml

```
@@ -32,7 +32,7 @@ requests = "^2.18"
 cachy = "^0.3.0"
 requests-toolbelt = "^0.9.1"
 cachecontrol = { version = "^0.12.4", extras = ["filecache"] }
-pkginfo = "^1.4"
+pkginfo = "^1.5"
```

```suggestion
pkginfo = "^1.6"
```
Poetry requires atleast 1.6.0.